src/harness/testcases/WINNF_FT_S_EXZ_testcase.py

```diff
@@ -0,0 +1,443 @@
+        'lowFrequency'] = 3645000000.0
+    grant_N1_3['operationParam']['operationFrequencyRange'][
+        'highFrequency'] = 3655000000.0
+
+    # Forming N2 grant request 
+    grant_N2_1 = json.load(
+          open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_N2_1['operationParam']['operationFrequencyRange'][
+        'lowFrequency'] = 3655000000.0
+    grant_N2_1['operationParam']['operationFrequencyRange'][
+        'highFrequency'] = 3650000000.0
+    grant_N2_2 = json.load(
+          open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_N2_2['operationParam']['operationFrequencyRange'][
+        'lowFrequency'] = 3655000000.0
+    grant_N2_2['operationParam']['operationFrequencyRange'][
+        'highFrequency'] = 3650000000.0
+
+
+    # Forming N3 grant request with overlapping frequency
+    grant_N3_1 = json.load(
+          open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_N3_1['operationParam']['operationFrequencyRange'][
+        'lowFrequency'] = 3645000000.0
+    grant_N3_1['operationParam']['operationFrequencyRange'][
+        'highFrequency'] = 3655000000.0
+    grant_N3_2 = json.load(
+          open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_N3_2['operationParam']['operationFrequencyRange'][
+        'lowFrequency'] = 3645000000.0
+    grant_N3_2['operationParam']['operationFrequencyRange'][
+        'highFrequency'] = 3655000000.0
+    grant_N3_3 = json.load(
+          open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_N3_3['operationParam']['operationFrequencyRange'][
+        'lowFrequency'] = 3645000000.0
+    grant_N3_3['operationParam']['operationFrequencyRange'][
+        'highFrequency'] = 3655000000.0
+ 
+    grants_N1 = [grant_N1_1, grant_N1_2, grant_N1_3]
+    grants_N2 = [grant_N2_1, grant_N2_2]
+    grants_N3 = [grant_N3_1, grant_N3_2, grant_N3_3]
+
+
+    # Create the actual config.
+    devices_N1 = [device_N1_1, device_N1_2, device_N1_3]
+    devices_N2 = [device_N2_1, device_N2_2]
+    devices_N3 = [device_N3_1, device_N3_2, device_N3_3]
+
+    config = {
+        'registrationRequests_N1' : devices_N1,
+        'registrationRequests_N2' : devices_N2,
+        'registrationRequests_N3' : devices_N3,
+        'grants_N1' : grants_N1,
+        'grants_N2' : grants_N2,
+        'grants_N3' : grants_N3
+    }
+    writeConfig(filename, config)
+
+  @configurable_testcase(generate_EXZ_2_default_config)
+  def test_WINNF_FT_S_EXZ_2(self,config_filename):
+    """Exclusion Zones defined in NTIA TR 15-517
+
+    The response should be SUCCESS for CBSDs located outside 50meters of all 
+    Exclusion Zones.
+    The responseCode = 400 for CBSDs located within 50meters of all Exclusion Zones 
+    or inside Exclusion Zones. 
+    """
+
+    # Enforce NTIA Exlusion zones
+    self._sas_admin.TriggerEnableNTIAExclusionZones()
+    config = loadConfig(config_filename)
```

Comment:
Please load the config first to be consistent with other test cases.

Reply:
Removed EXZ.2 from this PR. We will submit another PR once EXZ.1 get approved. We will code this similar to approved EXZ.1